src/components/Dropdown.tsx

```diff
@@ -0,0 +1,36 @@
+import { Icon } from "@material-ui/core";
+import React, { ReactElement } from 'react';
+
+import { styled } from '../styles';
+
+import { IconButton } from './IconButton';
+
+const DropdownWrapper = styled.div`
+  width: 200px;
+  position: absolute;
+  top: 80px;
+  right: 150px;
+  border: 1px solid;
+  border-radius: 8px;
+  background-color: ${ ({ theme }) => theme.colors.backgroundGray };
+`;
+
+interface IDropdownProps {
+  items: Array<{text: string, icon: ReactElement, onClick: ()=>void}>;
```

Comment:
would be better to create a separate type for the item. call it IDropdownItem, and pass here as IDropdownItem[]